### PR TITLE
Ignoring .git directorys while initalizing a fresh project

### PR DESCRIPTION
**Pull request by RoeHH**
With this change, the fresh init command would ignore .git directories

### CODE REVIEW

**Comment by RoeHH** [comment]
@lucacasonato open for review 😸

File: src/cli/init.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ async function init(directory: string) {
 
   try {
     const dir = [...Deno.readDirSync(directory)];
-    if (dir.length > 0) {
+    if (dir.length > 0 && !confirm("This is no Empty directory, some Files could get deleted, do you want to continue?")) {
```
Comment (lucacasonato) [inline src/cli/init.ts]:
```suggestion
    if (dir.length > 0 && !confirm("The target directory is not empty (files could get overwritten). Do you want to continue anyway?")) {
```

File: src/cli/init.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ async function init(directory: string) {
 
   try {
     const dir = [...Deno.readDirSync(directory)];
-    if (dir.length > 0) {
+    if (dir.length > 0 && !confirm("The target directory is not empty (files could get overwritten). Do you want to continue anyway?")) {
```
Comment (lucacasonato) [inline src/cli/init.ts]:
```suggestion
    if (
      dir.length > 0 &&
      !confirm(
        "This is no Empty directory, some Files could get deleted, do you want to continue?",
      )
    ) {
```

**Review by lucacasonato — COMMENTED** [review]
Thanks @RoeHH!

**Review by lucacasonato — APPROVED** [review]

**Pull request merged**